src/components/svg/LinkSvg.tsx

```diff
@@ -0,0 +1,36 @@
+import * as React from 'react';
+import { ISvgProps, Svg } from './Svg';
+import * as classname from 'classnames';
+import { ITooltipProps, Tooltip } from '../tooltip/Tooltip';
+
+export interface ILink {
+  url: string;
+  target?: string;
+}
+
+export interface ILinkSvgProps extends React.ClassAttributes<LinkSvg> {
+  link: ILink;
+  linkClasses?: string[];
+  svg?: ISvgProps;
+  tooltip?: ITooltipProps;
+}
+
+export class LinkSvg extends React.Component<ILinkSvgProps, {}> {
+  static defaultProps: Partial<ILinkSvgProps> = {
+    link: {
+      url: '',
+      target: '_blank',
+    },
+    linkClasses: [],
+    svg: { svgName: 'help', svgClass: `icon` },
```

Comment:
same thing for the SVG, it should probably be 2 properties instead of an object

Reply:
I think for the SVG its better to use the Interface created by the `<SVG />`. This is why a added the svg option like an object